simp_le.py

```diff
@@ -765,12 +765,12 @@ def _setup_logging(verbose):
 
 def _main(cli_args):
     args = create_parser().parse_args(cli_args)
+    _setup_logging(args.verbose)
```

Comment:
I think we actually don't want it to be called in `--test` case. Can you please change it to:

```python
if args.test:
  ...
_setup...
if args.vhosts ...
```